### PR TITLE
Add files

### PR DESCRIPTION
**Pull request by Intia-JoannaMarie**
Adding my folder that is needed for main branch

### CODE REVIEW

File: intia_joannamarie/script.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+function onTextChange() {
+    let name = document.querySelector("#name").value.trim();
+    let message = document.querySelector("#message").value.trim();
+    let commentButton = document.querySelector("#comment_button");
+
+    if (message.length > 0 && name.length > 0) {
```
Comment (despe-reymondchristian) [inline intia_joannamarie/script.js]:
```suggestion
    if (message.length && name.length ) {
```

**Review by Lopez-JhonLay — COMMENTED** [review]
nice code

**Review by despe-reymondchristian — COMMENTED** [review]
see suggested changes

File: intia_joannamarie/script.js (hunk 1 above)
Comment (Intia-JoannaMarie) [reply intia_joannamarie/script.js]: Got it

**Pull request merged**